app/src/main/res/layout/main_activity.xml

```diff
@@ -7,7 +7,7 @@
     android:layout_height="match_parent"
     android:background="@color/colorPrimaryDark"
     tools:context=".MainActivity">
-
+    // May be Better to put hint for which scrren & location is being occupied
```

Comment:
Discussion does not belong in source code. I called this out in the instructions, as it is a no-no.  It clutters up code.  Conversations are OK, but save them for the pull request.